tests/integration_tests/performance/test_block_performance.py

```diff
@@ -393,3 +395,81 @@ def test_block_vhost_user_performance(
         fio_block_size,
         vm,
     )
+
+
+@pytest.mark.nonci
+@pytest.mark.parametrize("vcpu_count", [1, 2], ids=["1vcpu", "2vcpu"])
```

Comment:
do you think we need both cases here?

Reply:
I took the reference from the other performance tests but initial run with this shows there is a small difference between metrics with 1 and 2 vcpus.